server/src/authority/authority.rs

```diff
@@ -901,7 +906,9 @@ impl Authority {
     ///  returned in the case of bad data, etc.
     pub fn update(&mut self, update: &Message) -> UpdateResult<bool> {
         // the spec says to authorize after prereqs, seems better to auth first.
+        #[cfg(feature = "dnssec")]
```

Comment:
I think we should just fail authorization in the no `dnssec` case, as opposed to not doing it... I'm ok with it always returning an error.